frontend/packages/console-shared/src/hooks/document-listener.ts

```diff
@@ -11,7 +11,14 @@ export enum KeyEventModes {
   FOCUS = 'FOCUS',
 }
 
-export const useDocumentListener = <T extends HTMLElement>(keyEventMap: KeyEventMap) => {
+const textInputKeyHandler = {
+  Escape: KeyEventModes.HIDE,
+  '/': KeyEventModes.FOCUS,
```

Comment:
We should be using the constant defined here:

https://github.com/openshift/console/blob/master/frontend/packages/console-shared/src/constants/common.ts#L16

Reply:
I had noticed that how about we make document-listener the central place for defining constants related to key bindings? The constant is only used in one other place and that is for namespace dropdown.

Reply:
I'm fine with that, just trying to avoid having it defined in two places.